mobile/src/services/notification/common/types/show-notification-params/notification-service.type.ts

```diff
@@ -0,0 +1,9 @@
+import { NotificationType } from '~/common/enums/enums';
+
+type ShowNotificationType = {
```

Comment:
Don't use Type for type names

```suggestion
type ShowNotificationParams = {
```